bindings/python/iree/rt/system_api_test.py

```diff
@@ -19,13 +19,14 @@
 
 from absl import logging
 from absl.testing import absltest
+import iree.compiler
+import iree.rt
 import numpy as np
-from pyiree import compiler
-from pyiree import rt
 
 
 def create_simple_mul_module():
-  binary = compiler.compile_str("""
+  binary = iree.compiler.compile_str(
+      """
```

Comment:
Nit: is this yapf formatting? Indenting here is weird... (in particular, the fact that the body here is less indented than the quotes)

Reply:
it is. I was able to make it look less weird by adding some extra white space to the string.